### PR TITLE
Little refactoring

### PR DESCRIPTION
**Pull request by joe513**

### CODE REVIEW

File: app/tests.py [hunk 1]
```diff
@@ -11,15 +11,14 @@
...
-        self.email = 'john@example.com'
-        self.password = 'dolphins'
+        self.username, self.password, self.email = 'John Doe', 'dolphins', 'john@example.com'
+
         self.user = User.objects.create_user(
             username=self.username,
             email=self.email,
             password=self.password
         )
-        self.client = Client()
```
Comment (joe513) [inline app/tests.py]: We don't have to provide own Client class. Please check [this](https://docs.djangoproject.com/en/2.0/topics/testing/tools/#django.test.SimpleTestCase.client)

**Comment by joe513** [comment]
It's very little refactor. I just thought it would be better to delete self.client. Also I don't have tasks enough but I want to code.

File: app/tests.py (hunk 1 above)
Comment (inoks) [reply app/tests.py]: We initialize client here and log with user credentials, so we can test authorized requests here.

**Comment by inoks** [comment]
Sorry, there no sense for me in that PR.
I will close it for now. Please feel free to update branch and sent new PR later.

File: app/tests.py (hunk 1 above)
Comment (joe513) [reply app/tests.py]: @inoks the client class is already in test case by default, we don't have to declare this. It's absolutely clear and ready to be referred for testing purposes. Please take a look at this short explanation https://docs.djangoproject.com/en/2.0/topics/testing/tools/#django.test.SimpleTestCase.client
and at this  https://docs.djangoproject.com/en/2.0/topics/testing/tools/#provided-test-case-classes (optional)
We don't have to create a new self.client by overriding above one.

**Pull request closed without merging**